### PR TITLE
2947_We_need_to_fix_some_remaining_Issues_on_collab_channel

### LINKED ISSUE #2939: Get error when click 'Sort' Owner on CollabStatusBoard
## 🐞 Bug description
Get error when click 'Sort' Owner on CollabStatusBoard 

## 🔢 Steps to Reproduce
1. Go to http://localhost:6611/?path=/docs/local-collabstatusboard--docs
2. Click on  'Sort' Owner
3. Notice the error

image

Note : It happens with storybook 8.x . We don't have this screen on 6.x before
- It also happens on gameplay.

## ⚙️ Expected behavior
Should enable to sort  'Owner' column on CollabStatusBoard 

## 🖥️ Screenscast

https://github.com/serge-web/serge/assets/107697044/5176cacc-85f3-46f1-a592-637f8b4cc730

## 💻 System information
- Browser : Chrome
- Browser Version : Version 120.0.6099.111

### LINKED ISSUE #2947: We need to fix some remaining Issues on collab channel
## 🐞 Bug description
We need to fix some remaining Issues on collab channel

## 🔢 Steps to Reproduce

1. Create a basic game with some force and some channels
2. Login from admin page
3. Click on dropdown of button 'Add Channel'
4. Select 'Collab Edit' to create first collab channel
5. Create permission for this channel 

image

6. Notice we have some issue below in wargame play

- Text 'Chat' and 'Response' are hard to see. And When message still haven't had response yet, it shows as the white long button. It makes user don't know what it is

image

Expected :  Text 'Chat' and 'Response' should be bold and white color as 'Feedback' text. And when message doesn't have response, we should not show that part.

- With this permission, Message with status 'InProgress' (Edit), user role 'CanSubmitForReview'should has button 'Submit for review', 'Save'

image

image

- Message with status 'InProgress' (Edit), user role 'CanApproved' should has button 'Save', 'Submit for review'

image

- Message with status 'InProgress' (Edit), user role 'CanRelease' should has button 'Save', 'Submit for review', 'Release'

image

- Message with status 'InProgress' (Edit), user role 'CanUnClaim' should has button 'Save', 'Submit for review', 'Release','Unclaim

image

- I think we have 2 status 'Pending' and 'Review' but now I see it has 'Pending Review'

image

image

=> What is the correct one? 

- The UI is quite messing and it's hard to follow the action

https://github.com/serge-web/serge/assets/107697044/5014ae6d-c6e6-444b-a10a-a682ea9225ee

- 'Assign' button is not stable. I have to try click many time to make it work.

https://github.com/serge-web/serge/assets/107697044/140e718d-fc7c-412a-b1d4-a5ebcf5a2cfc

- No action happens when approve for message which in status 'Pending Review'

https://github.com/serge-web/serge/assets/107697044/95e855ab-2e26-4316-bb73-315488237ba7

### PR DESCRIPTION
**Pull request by lilitkarapetyan**
Fixes #2947 
Fixes #2939 

## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

**Comment by lilitkarapetyan** [comment]
fixes #2947

**Comment by IanMayo** [comment]
Is this ready to go @Tristina1788 ?  Does it fix the problems in the linked issues?

**Comment by Tristina1788** [comment]
> Is this ready to go @Tristina1788 ? Does it fix the problems in the linked issues?

@IanMayo We still have these issues on this https://github.com/serge-web/serge/issues/2947#issuecomment-2173454617 .  Could you help to confirm we should fix these issues on this PR or not?

**Comment by IanMayo** [comment]
> > Is this ready to go @Tristina1788 ? Does it fix the problems in the linked issues?
> 
> @IanMayo We still have these issues on this [#2947 (comment)](https://github.com/serge-web/serge/issues/2947#issuecomment-2173454617) . Could you help to confirm we should fix these issues on this PR or not?

Yes @Tristina1788 - those are valid issues, that I would like fixed in this PR, please @lilitkarapetyan  

Except: 
> And in 'Terminology' part, I can't select value of Request Changes / Approve/ Release. Is this ok for now?

It is ok to delete and re-add the modified value, instead of needing to edit it.

**Comment by IanMayo** [comment]
@Tristina1788 - can we have a final check of this PR please?

**Comment by Tristina1788** [comment]
> @Tristina1788 - can we have a final check of this PR please?

@IanMayo  I checked and comment here : https://github.com/serge-web/serge/issues/2947#issuecomment-2238735606

**Review by IanMayo — APPROVED** [review]
Tested and working.
